notes/bugfix-21323

```diff
@@ -0,0 +1 @@
+correction to syntax for DG2's new props
```

Comment:
Thank you @jameshale . Two issues here:

1. The text should start with a `#` to make our release notes parser happy:

`# Correction to syntax for DG2's new props`

2. The filename should be renamed to `notes/bugfix-21323.md`